solr/core/src/test-files/solr/configsets/upload/with-lib-directive/solrconfig.xml

```diff
@@ -37,7 +37,7 @@
 
   <luceneMatchVersion>${tests.luceneMatchVersion:LATEST}</luceneMatchVersion>
 
-  <lib dir="${solr.install.dir:../../../..}/dist/" regex="solr-ltr-\d.*\.jar" />
+  <lib dir="${solr.install.dir:../../../..}/contrib/ltr/lib/" regex=".*\.jar" />
```

Comment:
Yea, people will have to update their `solrconfig.xml` lib. But it is more logical with one include for the contribs that have other dependency libs in 'lib/'. Or perhaps they will start using `SOLR_MODULES=ltr` instead :)

Reply:
Yeah still need to include this in the major-changed-in-solr-9 doc (or whatever it's called).